app/controllers/application_controller.rb

```diff
@@ -237,6 +237,7 @@ def after_sign_in_path_for(_user)
     return login_piv_cac_recommended_path if user_recommended_for_piv_cac?
     return second_mfa_reminder_url if user_needs_second_mfa_reminder?
     return sp_session_request_url_with_updated_params if sp_session.key?(:request_url)
+    return user_password_compromised_url if session[:redirect_to_password_compromised].present?
```

Comment:
This seems pretty late in the set of checks. Specifically, isn't `sp_session_request_url_with_updated_params` the SP sign-in path, so a user would never see this when signing in to an SP? (which is vast majority of sign-ins)

Reply:
Oh good call, I will move it up.